Mayden-Academy/AcademyPortal
# S2 t8 sortable fe

Requirements:
- added Sortable.js file
- added stageReorder.js to create a batch edit package when the list is reordered. Calls the sendEditRequest function in stagesPage.js
- added classes to table body and table rows to make Sortable work, and added id to table rows.

To test:
- drag to reorder records in the browser, check the db to see if the order updates correctly
- clear browser cache, and reload page to check the new order persists
